dynaconf/loaders/vault_loader.py

```diff
@@ -130,3 +130,17 @@ def write(obj, data=None, **kwargs):
     path = "/".join([obj.VAULT_PATH_FOR_DYNACONF, obj.current_env.lower()])
     client.secrets.kv.create_or_update_secret(path, secret=data)
     load(obj)
+
+
+def load_keys(obj, path):
+    """load all the keys specified at gived path in vault.
+
+    :param obj: settings object
+    :param path: path to the vault secrets
+    :return: list containing all the keys at the given path
+    """
```

Comment:
Please add some lines to docstring stating the reason for that function to exist and provide a use case, so in future we will know why that function exists as it will not be in use by dynaconf itself.

```suggestion
    ...
    This function is a helper to get a list of all the existing envs in the source of data, 
    the use case is:
        existing_envs = vault_loader.list_envs(settings)
        for env in exiting_envs:
            with settings.using_env(env):  # switch to the env
                # do something with a key of that env
        
    """
```

Reply:
Sure thing.